README.md

```diff
@@ -45,7 +43,7 @@ You can find the specific version of Kyuubi documentation as listed below.
 - [v1.0.1](https://kyuubi.readthedocs.io/en/v1.0.1/)
 - [v1.0.0](https://kyuubi.readthedocs.io/en/v1.0.0/)
 
-For 0.8 and earlier versions, please check the [Github Pages](https://yaooqinn.github.io/kyuubi/) directly.
+For 0.8 and earlier versions, please check the [Github Pages](https://NetEase.github.io/kyuubi/) directly.
```

Comment:
can you check this still work?

Reply:
![image](https://user-images.githubusercontent.com/6757692/114838597-6caa6c80-9e07-11eb-88c1-4d1f544fde12.png)